pkg/collector/client.go

```diff
@@ -32,46 +60,104 @@ func (c *clientCollector) Collect(ch chan<- prometheus.Metric) {
 		return
 	}
 
-	ch <- prometheus.MustNewConstMetric(c.omadaClientConnectedTotal, prometheus.GaugeValue, float64(len(clients)),
-		site, client.SiteId)
+	totals := map[string]int{}
 
 	for _, item := range clients {
 		vlanId := fmt.Sprintf("%.0f", item.VlanId)
 		port := fmt.Sprintf("%.0f", item.Port)
+		connectionModeFmt := ""
 		if item.Wireless {
 			wifiMode := fmt.Sprintf("%.0f", item.WifiMode)
+			connectionModeFmt = FormatWifiMode(int(item.WifiMode))
```

Comment:
If we modify the `wifiMode` variable like the comment above we can just set this to `connectionModeFmt = "wireless"`. Then we can easily filter on total wireless clients as well as each individual protocol

Reply:
How is it different than whats already happening?